tests/test_influxdbreporting.py

```diff
@@ -184,10 +184,10 @@ async def test_available_metrics(server):
                     base_types = [base_type]
                 types[key] = base_types
 
-    assert metrics["db.connected,component=server"]["value"]
-    assert "db.max_pool,component=server" in metrics
-    assert "db.open_connections,component=server" in metrics
-    assert "db.free_connections,component=server" in metrics
```

Comment:
Why is the `component=server` part gone here?

Reply:
it is moved to the output reporter, so in the output, it is there, internally, it is not